### PR TITLE
Fixed for Windows.

### LINKED ISSUE #122: Produce and test windows builds
Kind of self explanatory.  May require a windows box but not sure (see #32 for details on that).

### PR DESCRIPTION
**Pull request by jcvernaleo**
Need to construct the path for Windows rather than using env variables
the way were we doing it before.

Move all functions that need path and config info to a single file,
config.js.

Detached should be false, esp for windows.

Don't use fd 4 that doesn't seem to work on win with wallet.

Closes #122

### CODE REVIEW

**Comment by jcvernaleo** [comment]
Waiting for it to actually fully sync before I want a review on this.

File: app/config.js [hunk 1]
```diff
@@ -49,3 +53,66 @@ export function getCfgPath() {
...
+
+export function getCert() {
+  var cert = '';
+  var cfg = getCfg();
+  if (cfg.cert_path != '') {
+    return(cfg.cert_path);
+  }
+  var certPath = '';
+  if (os.platform() == 'win32') {
+    certPath = path.join(os.homedir(), 'AppData', 'Local', 'Decrediton', 'rpc.cert');
```
Comment (jrick) [inline app/config.js]: probably want `process.env.LOCALAPPDATA`.

Comment (jcvernaleo) [inline app/config.js]: That is what I originally had and it (and process.env.APPDATA) were both undefined when I ran it in the electron app (but had the value I expected with standalone node).

Comment (jrick) [inline app/config.js]: oh interesting, so that env var is missing when in production mode? :/

Comment (jcvernaleo) [inline app/config.js]: It does seem that way (which is super annoying).

Comment (jrick) [inline app/config.js]: could use a comment here if that's the case so others know why it was done this way

Comment (jcvernaleo) [inline app/config.js]: comment added

**Comment by jcvernaleo** [comment]
Okay, it synced up all the way for me.  So @alexlyp or @jrick feel free to have at it.

**Pull request merged**